test/e2e/utils_test.go

```diff
@@ -324,3 +324,26 @@ func httpGetResponseContains(t *testing.T, url string, client *http.Client, subs
 
 	return strings.Contains(string(body), substring)
 }
+
+// getKubernetesLogs gets logs af ALL containers inside pod.
+func getKubernetesLogs(ctx context.Context, t *testing.T, env environments.Environment, namespace string, name string) (string, error) {
```

Comment:
If I may suggest: let's pick one strategy for handling errors:

- `error` return value
- using `*testing.T` in assertions

Mixing those 2 feels unnecessarily complicated.

Reply:
There ARE some differences for the two errors:
- errors using `t.Require` means that we should fail the test immediately, e.g: failed to create kubeconfig file
- errors returned means that the error should be processed by the caller, e.g: the pod is not started yet and we cannot call `logs` to it.

Reply:
We might want to comment this as it's not obvious what the `t` is going to be used for.

Additionally

- It would be nice to mention what does the return value represent, i.e. that it's a `string` containing all container logs from that pods (if I get this right)
- Minor renaming suggestions:

```suggestion
func getPodLogs(ctx context.Context, t *testing.T, env environments.Environment, namespace string, podName string) (string, error) {
```

Reply:
renamed.